packages/web-components/src/wallet-connection/watchWallet.js

```diff
@@ -65,33 +64,28 @@ export const watchWallet = async (chainStorageWatcher, address) => {
     ),
   );
 
-  // NB: this watches '.current' but only notifies of changes to offerToPublicSubscriberPaths
-  await /** @type {Promise<void>} */ (
-    new Promise((res, rej) => {
-      let lastPaths;
-      chainStorageWatcher.watchLatest(
-        ['data', `published.wallet.${address}.current`],
-        value => {
-          res();
-          const { offerToPublicSubscriberPaths: currentPaths } = value;
-          if (currentPaths === lastPaths) return;
-
-          publicSubscriberPathsNotifierKit.updater.updateState(
-            harden(currentPaths),
-          );
-        },
-        err => {
-          if (!lastPaths) {
-            rej();
-          } else {
-            throw Error(err);
-          }
-        },
+  const smartWalletStatusNotifierKit = {};
```

Comment:
Can we make this an actual notifier? It would make it so the dapp can know when the smart wallet is successfully provisioned. I would make the change myself but I want to be sure there's no issues with your intended use case.
  
  ```
  const smartWalletStatusNotifierKit = makeNotifierKit(
    /** @type { {provisioned: boolean} | null } */ (
      null
    ),
  );
  ```